diff -Nr -U 0 release/1.7/app/admin.py release/1.3/app/admin.py
--- release/1.7/app/admin.py	2016-06-16 03:36:26.844210459 -0300
+++ release/1.3/app/admin.py	1969-12-31 21:00:00.000000000 -0300
@@ -1,3 +0,0 @@
-from django.contrib import admin
-
-# Register your models here.
diff -Nr -U 0 release/1.7/app/tests.py release/1.3/app/tests.py
--- release/1.7/app/tests.py	2016-06-16 03:36:26.844210459 -0300
+++ release/1.3/app/tests.py	2016-06-16 03:36:26.844210459 -0300
@@ -0,0 +1,7 @@
+"""
+This file demonstrates writing tests using the unittest module. These will pass
+when you run "manage.py test".
+
+Replace this with more appropriate tests for your application.
+"""
+
@@ -3 +10,7 @@
-# Create your tests here.
+
+class SimpleTest(TestCase):
+    def test_basic_addition(self):
+        """
+        Tests that 1 + 1 always equals 2.
+        """
+        self.assertEqual(1 + 1, 2)
diff -Nr -U 0 release/1.7/app/views.py release/1.3/app/views.py
--- release/1.7/app/views.py	2016-06-16 03:36:26.844210459 -0300
+++ release/1.3/app/views.py	2016-06-16 03:36:26.844210459 -0300
@@ -1,2 +0,0 @@
-from django.shortcuts import render
-
