--- release/1.3/default_settings.py	2016-06-16 03:36:26.844210459 -0300
+++ release/1.3/startproject_settings.py	2016-06-16 03:36:26.844210459 -0300
@@ -21 +21 @@
-DATABASES = {}
+DATABASES = {'default': {'ENGINE': 'django.db.backends.', 'NAME': '', 'HOST': '', 'USER': '', 'PASSWORD': '', 'PORT': ''}}
@@ -34 +34 @@
-DEBUG = False
+DEBUG = True
@@ -62 +62 @@
-INSTALLED_APPS = []
+INSTALLED_APPS = ['django.contrib.auth', 'django.contrib.contenttypes', 'django.contrib.sessions', 'django.contrib.sites', 'django.contrib.messages', 'django.contrib.staticfiles']
@@ -83,0 +84 @@
+ROOT_URLCONF = 'project.urls'  ###
@@ -97 +98 @@
-SETTINGS_MODULE = 'project.defaults'  ###
+SETTINGS_MODULE = 'project.settings'  ###
@@ -99,0 +101 @@
+SITE_ID = 1  ###
@@ -104 +106 @@
-STATIC_URL = None
+STATIC_URL = '/static/'
@@ -106 +108 @@
-TEMPLATE_DEBUG = False
+TEMPLATE_DEBUG = True
@@ -122 +124 @@
-USE_L10N = False
+USE_L10N = True
