--- release/1.6/default_settings.py	2016-06-16 01:06:42.816211531 -0300
+++ release/1.6/startproject_settings.py	2016-06-16 01:06:42.664211531 -0300
@@ -8,0 +9 @@
+BASE_DIR = '.'  ###
@@ -20 +21 @@
-DATABASES = {}
+DATABASES = {'default': {'ENGINE': 'django.db.backends.sqlite3', 'AUTOCOMMIT': True, 'ATOMIC_REQUESTS': False, 'NAME': './db.sqlite3', 'TEST_MIRROR': None, 'CONN_MAX_AGE': 0, 'TEST_NAME': None, 'TIME_ZONE': 'UTC', 'TEST_COLLATION': None, 'PORT': '', 'HOST': '', 'USER': '', 'TEST_CHARSET': None, 'PASSWORD': '', 'OPTIONS': {}}}
@@ -26 +27 @@
-DEBUG = False
+DEBUG = True
@@ -54 +55 @@
-INSTALLED_APPS = ()
+INSTALLED_APPS = ('django.contrib.admin', 'django.contrib.auth', 'django.contrib.contenttypes', 'django.contrib.sessions', 'django.contrib.messages', 'django.contrib.staticfiles')
@@ -70 +71 @@
-MIDDLEWARE_CLASSES = ('django.middleware.common.CommonMiddleware', 'django.contrib.sessions.middleware.SessionMiddleware', 'django.middleware.csrf.CsrfViewMiddleware', 'django.contrib.auth.middleware.AuthenticationMiddleware', 'django.contrib.messages.middleware.MessageMiddleware')
+MIDDLEWARE_CLASSES = ('django.contrib.sessions.middleware.SessionMiddleware', 'django.middleware.common.CommonMiddleware', 'django.middleware.csrf.CsrfViewMiddleware', 'django.contrib.auth.middleware.AuthenticationMiddleware', 'django.contrib.messages.middleware.MessageMiddleware', 'django.middleware.clickjacking.XFrameOptionsMiddleware')
@@ -76,0 +78 @@
+ROOT_URLCONF = 'project.urls'  ###
@@ -93 +95 @@
-SETTINGS_MODULE = 'project.defaults'  ###
+SETTINGS_MODULE = 'project.settings'  ###
@@ -101 +103 @@
-STATIC_URL = None
+STATIC_URL = '/static/'
@@ -103 +105 @@
-TEMPLATE_DEBUG = False
+TEMPLATE_DEBUG = True
@@ -111 +113 @@
-TIME_ZONE = 'America/Chicago'
+TIME_ZONE = 'UTC'
@@ -115 +117 @@
-USE_L10N = False
+USE_L10N = True
@@ -117 +119 @@
-USE_TZ = False
+USE_TZ = True
@@ -119 +121 @@
-WSGI_APPLICATION = None
+WSGI_APPLICATION = 'project.wsgi.application'
